backend/tournesol/views/rate_later.py

```diff
@@ -93,6 +107,29 @@ def get_serializer(self, *args, **kwargs):
 
     def perform_create(self, serializer):
         rate_later_instances = serializer.save()
+
+        if self.request.query_params.get("entity_seen", "false") == "true":
+            contributor_ratings = [
+                ContributorRating(
+                    poll_id=self.poll_from_url.id,
+                    user_id=self.request.user.id,
+                    entity_id=rate_later["entity"]["pk"],
+                    is_public=True,
+                    entity_seen=True,
+                )
+                for rate_later in serializer.validated_data
+            ]
+
+            # Let's remember that signals won't be triggered by `bulk_create`
+            # and the .save() method of ContributorRating won't be called.
+            ContributorRating.objects.bulk_create(
+                contributor_ratings,
+                update_conflicts=True,
+                update_fields=["entity_seen"],
+                unique_fields=["poll_id", "user_id", "entity_id"],
+            )
+
+        # TOFIX: rate_later_instances seems to always be an empty list
         for rate_later in rate_later_instances:
             self.prefetch_entity(rate_later)
```

Comment:
Hum, how did you observe that  `rate_later_instances` is an empty list?

Not necessarily a topic for this PR, by maybe we could add a test to validate the number of SQL queries when submitting a batch, and the prefetch works as expected?